lib/jekyll/site.rb

```diff
@@ -45,10 +45,12 @@ def config=(config)
       @config = config.clone
 
       %w(safe lsi highlighter baseurl exclude include future unpublished
-        show_drafts limit_posts keep_files gems).each do |opt|
+        show_drafts limit_posts keep_files).each do |opt|
         self.send("#{opt}=", config[opt])
       end
 
+      self.gems = config["plugins"]
```

Comment:
I suspect this is here because we want to maintain the Ruby API even if the user-facing configuration key is changing? If so, I'd add a quick line comment here so a well-meaning developer doesn't "optimize" this in the future.